buchschloss/config/main.py

```diff
@@ -209,8 +222,11 @@ def merge_ui(config):
 
 def insert_name_data(config):
     """load name data into [utils][names]"""
-    config['utils']['names'] = load_names(
+    name_data = load_names(
         config['utils']['names']['file'], config['utils']['names']['format'])
+    # without unrepr=True, this gets converted to a Section
+    # which fails with numeric keys (level representations)
+    config['utils'].__setitem__('names', name_data, unrepr=True)
```

Comment:
how about converting keys to strings on access?

Reply:
problematic with gui2's level selection widget (``(mtk.OptionChoiceWidget,  {'values': list(utils.level_names.items()), 'default': 1})``)